### PR TITLE
add babel-core@5.6.15 with npm auto-update

### PR DESCRIPTION
**Pull request by maruilian11**

### CODE REVIEW

File: ajax/libs/babel-core/package.json [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    "resolve": "^1.1.6",
+    "shebang-regex": "^1.0.0",
+    "slash": "^1.0.0",
+    "source-map": "^0.4.0",
+    "source-map-support": "^0.2.10",
+    "strip-json-comments": "^1.0.2",
+    "to-fast-properties": "^1.0.0",
+    "trim-right": "^1.0.0"
+  },
+  "npmName": "jquery",
```
Comment (jmm) [inline ajax/libs/babel-core/package.json]: Is this supposed to be `babel-core`? I noticed that the `babel-core` version is from a couple of months ago (`v5.6.15`) even though it says `npm autoupdate enabled`  and I'm wondering if this is why.

Comment (PeterDaveHello) [inline ajax/libs/babel-core/package.json]: @jmm, you are correct, sorry for the wrong config and thanks for your report. I'm going to fix it now.

Comment (PeterDaveHello) [inline ajax/libs/babel-core/package.json]: @jmm babel-core v5.6.18~5.8.23 been added now, thanks again!
https://github.com/cdnjs/cdnjs/commit/68c40decde19e7d0035c2e1b81d17c41610b9728

**Comment by jmm** [comment]
@PeterDaveHello Awesome, thank you!

**Comment by PeterDaveHello** [comment]
My pleasure.

**Pull request merged**